helper/calculateNetworth.js

```diff
@@ -27,7 +27,7 @@ const calculateNetworth = (items, purseBalance, bankBalance, prices, onlyNetwort
     }
 
     // Sort items by price
-    if (!onlyNetworth && categories[category].items.length > 0) {
+    if (sortData && !onlyNetworth && categories[category].items.length > 0) {
```

Comment:
This also disables combining items like if you have 10 stacks of enchanted hay bales it will shows each stack instead of combining. See `.reduce` below.